src/containerapp/azext_containerapp/custom.py

```diff
@@ -2408,3 +2409,74 @@ def delete_environment_telemetry_app_insights(cmd,
     
     return r
 
+
+def add_environment_telemetry_otlp(cmd,
+                                   name,
+                                   resource_group_name,
+                                   otlp_name,
+                                   endpoint,
+                                   insecure=False,
+                                   enable_open_telemetry_traces=False,
```

Comment:
Why the default value for `enable_open_telemetry_traces` and `enable_open_telemetry_logs` and `enable_open_telemetry_metrics` here is `False`?

It should be `None`, otherwise you cannot distinguish if the user wants to disable it.

Reply:
offline synced